ownCloud/Client/Actions/Actions+Extensions/PresentationModeAction.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  PresentationModeAction.swift
+//  ownCloud
+//
+//  Created by Matthias Hühne on 12.02.21.
+//  Copyright © 2019 ownCloud GmbH. All rights reserved.
+//
+
+/*
+ * Copyright (C) 2021, ownCloud GmbH.
+ *
+ * This code is covered by the GNU Public License Version 3.
+ *
+ * For distribution utilizing Apple mechanisms please see https://owncloud.org/contribute/iOS-license-exception/
+ * You should have received a copy of this license along with this program. If not, see <http://www.gnu.org/licenses/gpl-3.0.en.html>.
+ *
+ */
+
+import UIKit
+import ownCloudSDK
+import ownCloudAppShared
+
+class PresentationModeAction: Action {
+	override class var identifier : OCExtensionIdentifier? { return OCExtensionIdentifier("com.owncloud.action.presentationmode") }
+	override class var category : ActionCategory? { return .normal }
+	override class var name : String? { return "Presentation Mode".localized }
+	override class var locations : [OCExtensionLocationIdentifier]? { return [.moreDetailItem, .keyboardShortcut] }
+	override class var keyCommand : String? { return "P" }
+	override class var keyModifierFlags: UIKeyModifierFlags? { return [.command, .alternate] }
+
+	// MARK: - Extension matching
+	override class func applicablePosition(forContext context: ActionContext) -> ActionPosition {
+		if DisplaySleepPreventer.shared.preventCount > 0, let hostViewController = context.viewController, !(hostViewController.navigationController?.isNavigationBarHidden ?? false) {
+			return .none
+		}
+
+		return .middle
+	}
+
+	// MARK: - Action implementation
+	override func run() {
+		guard context.items.count > 0, let hostViewController = context.viewController else {
+			completed(with: NSError(ocError: .insufficientParameters))
+			return
+		}
+
+		let alertController = UIAlertController(title: "Presentation Mode".localized, message: "Enabling presentation mode will prevent the display from sleep mode until the view is dismissed.".localized, preferredStyle: .alert)
```

Comment:
```suggestion
		let alertController = UIAlertController(title: "Presentation Mode".localized, message: "Enabling presentation mode will prevent the display from sleep mode until the view is closed.".localized, preferredStyle: .alert)
```

See comment in `changelog/unreleased/704`

Reply:
Fixed wording!